cinema/views.py

```diff
@@ -54,6 +77,32 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def get_queryset(self):
+        queryset = MovieSession.objects.all()
+        date_ = self.request.query_params.get("date")
+        movie = self.request.query_params.get("movie")
+        if date_ and movie:
+            queryset = queryset.filter(
+                Q(show_time__date=date_) &
+                Q(movie_id=movie)
+            )
+        elif date_:
+            queryset = queryset.filter(show_time__date=date_)
+        elif movie:
+            queryset = queryset.filter(movie_id=movie)
```

Comment:
Bad approach. What if you had 10 params? It is much better to make a separate `if` statement for every param.